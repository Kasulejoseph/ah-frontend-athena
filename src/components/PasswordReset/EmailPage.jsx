import React from 'react'
import "./PasswordReset.scss";

export const EmailPage = () => (
  <div className="w-100 flex flex-center flex-column align-items-center justify-content-center">
    <h2 className="blue-text">An email has been sent</h2>
    <h4 className="">
      Please check your email and follow the 
      instructions to reset your passsword
    </h4>
  </div>
  );


export default EmailPage;
