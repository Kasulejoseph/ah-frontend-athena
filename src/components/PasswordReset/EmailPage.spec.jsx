import React from 'react';
import { shallow } from 'enzyme';
import {EmailPage} from './EmailPage';

describe('test email sent page', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<EmailPage />)
  })
  it('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
  it('test whether the wrapper is defined', () => {
    expect(wrapper).toBeDefined()
  })
  it('should find the h2 tag', () => {
    expect(wrapper.find('h2').text()).toBe('An email has been sent')
  });
});