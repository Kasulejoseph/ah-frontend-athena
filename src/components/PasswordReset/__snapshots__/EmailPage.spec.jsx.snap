// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test email sent page matches snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <EmailPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2
          className="blue-text"
        >
          An email has been sent
        </h2>,
        <h4
          className=""
        >
          Please check your email and follow the instructions to reset your passsword
        </h4>,
      ],
      "className": "w-100 flex flex-center flex-column align-items-center justify-content-center",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "An email has been sent",
          "className": "blue-text",
        },
        "ref": null,
        "rendered": "An email has been sent",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Please check your email and follow the instructions to reset your passsword",
          "className": "",
        },
        "ref": null,
        "rendered": "Please check your email and follow the instructions to reset your passsword",
        "type": "h4",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2
            className="blue-text"
          >
            An email has been sent
          </h2>,
          <h4
            className=""
          >
            Please check your email and follow the instructions to reset your passsword
          </h4>,
        ],
        "className": "w-100 flex flex-center flex-column align-items-center justify-content-center",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "An email has been sent",
            "className": "blue-text",
          },
          "ref": null,
          "rendered": "An email has been sent",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Please check your email and follow the instructions to reset your passsword",
            "className": "",
          },
          "ref": null,
          "rendered": "Please check your email and follow the instructions to reset your passsword",
          "type": "h4",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
